src/app/services/comment.service.ts

```diff
@@ -43,13 +47,15 @@ export class CommentService {
     await firebase.database().ref(`/blog/users/${userId}`).set(user);
   }
 //finds all comments based in the parentId of the comment
-  getCommentsByParentId(parentId, type) {
+  commentsByParentIdRef(parentId, type) {
```

Comment:
I'm going to be OCD here too and suggest a different order and simpler form: "getCommentsRef()"